packages/ui/components/form/inputs/Input.tsx

```diff
@@ -39,174 +18,6 @@ export function InputLeading(props: JSX.IntrinsicElements["div"]) {
   );
 }
 
-type InputFieldProps = {
-  label?: ReactNode;
-  LockedIcon?: React.ReactNode;
-  hint?: ReactNode;
-  hintErrors?: string[];
-  addOnLeading?: ReactNode;
-  addOnSuffix?: ReactNode;
-  inputIsFullWidth?: boolean;
-  addOnFilled?: boolean;
-  addOnClassname?: string;
-  error?: string;
-  labelSrOnly?: boolean;
-  containerClassName?: string;
-  showAsteriskIndicator?: boolean;
-  t?: (key: string) => string;
-} & React.ComponentProps<typeof Input> & {
-    labelProps?: React.ComponentProps<typeof Label>;
-    labelClassName?: string;
-  };
-
-type AddonProps = {
-  children: React.ReactNode;
-  isFilled?: boolean;
-  className?: string;
-  error?: boolean;
-};
-
-const Addon = ({ isFilled, children, className, error }: AddonProps) => (
-  <div
-    className={classNames(
-      "addon-wrapper border-default [input:hover_+_&]:border-emphasis [input:hover_+_&]:border-l-default [&:has(+_input:hover)]:border-emphasis [&:has(+_input:hover)]:border-r-default h-9 border px-3",
-      isFilled && "bg-subtle",
-      className
-    )}>
-    <div
-      className={classNames(
-        "min-h-9 flex flex-col justify-center text-sm leading-7",
-        error ? "text-error" : "text-default"
-      )}>
-      <span className="flex whitespace-nowrap">{children}</span>
-    </div>
-  </div>
-);
-
-export const InputField = forwardRef<HTMLInputElement, InputFieldProps>(function InputField(props, ref) {
```

Comment:
Webpack not able to do tree shaking for this module very well, so move the components in separate files.